app/config/services.yml

```diff
@@ -4,6 +4,15 @@ parameters:
     #parameter_name: value
 
 services:
-    #service_name:
-    #    class: AppBundle\Directory\ClassName
-    #    arguments: ['@another_service_name', 'plain_value', '%parameter_name%']
+    _defaults:
+        autowire: true
+        autoconfigure: true
+        public: false
+
+    AppBundle\:
+        resource: '../../src/AppBundle/{Command,Form,EventSubscriber,Twig,Voter}'
```

Comment:
Well, I thought that `%kernel.project_dir%` was rather clear, because:
* I don't have to think where is the `file.yml` placed to write relatived paths (thinking in new configurations),
* Also, it account with IDE auto-completation (may be more hard for those who don't have it),
* Beside, it would allows us move these files without to break its internal configuration (i.e relative paths are fragile into **flex**ible structures), 

But I could be wrong and it's not enough of course :)